heracal/omni.py

```diff
@@ -219,8 +219,40 @@ def compute_reds(nant, pols, *args, **kwargs):
             reds += [[(Antpol(i, pi, nant), Antpol(j, pj, nant)) for i, j in gp] for gp in _reds]
     return reds
 
-
-def aa_to_info(aa, pols=['x'], fcal=False, **kwargs):
+def minVreds(reds):
```

Comment:
I don't understand the name of this function. Minimal .. Stokes V ... redundancies? A longer, more informative name would be helpful.